tests/gis_tests/gis_migrations/test_operations.py

```diff
@@ -41,15 +45,24 @@ def apply_operations(self, app_label, project_state, operations):
         with connection.schema_editor() as editor:
             return migration.apply(project_state, editor)
 
-    def set_up_test_model(self):
+    def set_up_test_model(self, force_raster_creation=False):
+        # Add regular fields to test model
+        testfields = [
+            ("id", models.AutoField(primary_key=True)),
+            ('name', models.CharField(max_length=100, unique=True)),
+        ]
+        # For GIS enabled DB add geometry field
+        if connection.features.gis_enabled:
```

Comment:
not sure why this check is needed -- don't these tests only run on `gis_enabled` backends?